app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.create(user_params)
```

Comment:
Brought in user information from new form and then if it was valid, which it was, it saved to the DB and did the proper redirection / showed the flash message on the profile page